scantools/tests/download_navvis_test_data.py

```diff
@@ -0,0 +1,32 @@
+import requests
+import hashlib
+import zipfile
+import os
+
+datasetUrl = "https://github.com/microsoft/lamar-benchmark/releases/download/v1.1/NavVisTestData.zip"
+expectedSha1 = "0a28991004e58d9dc51167ea23f9f2ba66b26ab5"
+zipFileName = "NavVisTestData.zip"
+
+print("Downloading NavVis test data...")
+
+if(os.path.exists(zipFileName)):
```

Comment:
nit: I would make that this script takes an output folder, and outside you create a temporal directory, something like:
`navvis_test_data_path = Path(tempfile.mkdtemp())`

Then you avoid check if exists, also you can remove the .gitignore. The idea is not to populate the git repo with untracked files.

Reply:
k will come back to this